CHANGELOG.md

```diff
@@ -122,6 +122,7 @@
 * [BUGFIX] Fix metadata API using wrong JSON field names. #7475
 * [BUGFIX] Ruler: fix native histogram recording rule result corruption. #7552
 * [BUGFIX] Rules: improve error handling when querier is local to the ruler. #7567
+* [BUGFIX] Querier: fix HTTP status code translations for remote read requests. previously endpoints would return either only HTTP 400 or HTTP 500 on any error. #7487
```

Comment:
```suggestion
* [BUGFIX] Querier: fix HTTP status code translations for remote read requests. Previously endpoints would return either only HTTP 400 or HTTP 500 on any error. #7487
```

Reply:
Should we also add something like this?

```
Currently remote-read has conflicting behaviours:
  - when returning samples all internal errors are translated to HTTP 400
  - when returning chunks all internal errors are translated to HTTP 500
```

Reply:
i included that in https://github.com/grafana/mimir/pull/7589/commits/53c4b3f8ad76564b55b8926669c5bfe1e699dddc, but tried to fit it in a single line. How does it look like?